internal/bminventory/inventory_test.go

```diff
@@ -4373,6 +4373,18 @@ var _ = Describe("GetClusterInstallConfig", func() {
 		Expect(config.BaseDomain).To(Equal("example.com"))
 	})
 
+	It("returns the correct default config", func() {
```

Comment:
I think this test will break if it is run in an environment with different defaults

Reply:
I don't see how it could happen. It is a unit-test, it should not be affected by anything.

Reply:
This comment was correct when this test used `os.Env` but now it's ok because we don't set envs in unit-tests. 
So i think that it can be resolved.